code/modules/hydroponics/grown/misc.dm

```diff
@@ -157,6 +157,26 @@
 	reagents.chem_temp = 1000 //Sets off the black powder
 	reagents.handle_reactions()
 
+//Murmuring Tree
+/obj/item/seeds/murmuring_wood
+	name = "pack of murmuring tree seeds"
+	desc = "These seeds grow into murmuring trees."
+	icon_state = "seed-murmuring"
+	species = "murmuring"
+	plantname = "Murmuring Tree"
+	product = /obj/item/reagent_containers/food/snacks/grown/shell/gatfruit
```

Comment:
i know, i know. this needs to change to the actual mech itself.

Reply:
Just a helpful pointer, [the application of .addjuice can help you modify the effects of your plant based on potency to achieve what you spoke about if you wish to use my own past attempt as a example](https://github.com/tgstation/tgstation/pull/26751/files#diff-a54f04b9463de9207e825007965b42ebR100). 

> if you don't have enough potency though, you're going to get the really lame treant instead

Reply:
Hey, great pointer. Right now it drops a big cocoon you smack open for the mech, I can have that be similar to what you linked